lib/utils.js

```diff
@@ -60,7 +60,15 @@ function parseOutput(lines, inputFilename, pathPrefix) {
     eachLine(lines, function (line) {
         line = line.split('<stdin>').join('<source>');
         if (pathPrefix) line = line.replace(pathPrefix, "");
-        if (inputFilename) line = line.split(inputFilename).join('<source>');
+        if (inputFilename && line.includes(inputFilename))
+            line = line.split(inputFilename).join('<source>');
+        else {
+            const changedFilename = /^(.+)\:\d+\:\d+\:/;
```

Comment:
I'm probably missing something - how does this avoid picking up things like system header filenames? If there's an error in /some/other/included/header.h:2:3 it seems this code will misattribute it to line 2 of the user's code?

Reply:
@mattgodbolt Whoops. Thanks, I'll look into it.